dietpi/dietpi-software

```diff
@@ -1071,6 +1071,16 @@ DietPi-Software will decrypt and use it for software installs. You can change it
 		aSOFTWARE_REQUIRES_SQLITE[$software_id]=1
 		# - Bullseye: python(2)-virtualenv not available (yet): https://packages.debian.org/python-virtualenv
 		aSOFTWARE_AVAIL_G_DISTRO[$software_id,6]=0
+		#------------------
+                software_id=183
+
+                aSOFTWARE_NAME[$software_id]='Bitwarden RS'
+                aSOFTWARE_DESC[$software_id]='Unoffical Bitwarden password manager server written in rust'
+                aSOFTWARE_TYPE[$software_id]=0
+                aSOFTWARE_CATEGORY_INDEX[$software_id]=4
+                aSOFTWARE_ONLINEDOC_URL[$software_id]='p=24799#p24799' #idk what to do here
```

Comment:
We need to create an online documentation page: https://dietpi.com/phpbb/viewtopic.php?p=28177#p28177
And here: https://github.com/MichaIng/DietPi-Docs/blob/dev/docs/software/cloud.md
I'll switch those URLs over to the new docs with an upcoming release.

Client setup and web vault access (when implemented) should be added, or rough instructions (or a link) about how to enable the web vault.